console.log("script end");
try {
    console.log(rahul);
}
catch(error) {
    console.log(error);
}
console.log("script end");
/* 
try-catch block only handles synchronous code nit scheduled like setTimeout, setInterval etc
*/
